draft-ietf-httpbis-sfbis.md

```diff
@@ -191,6 +191,11 @@ being used.</t>
 </artwork>
 </blockquote>
 
+Note that a field is defined by referencing a specific RFC for Structured Fields, and the types available for use in its value are limited to those defined in that RFC. For example, a field whose definition references this document can have a value that uses the Date type ({{date}}), whereas a field whose definition references RFC8941 cannot, because it will be treated as invalid (and likely discarded) by implementations of that specification.
```

Comment:
```suggestion
Note that a field is defined by referencing a specific RFC for Structured Fields, and the types available for use in its value are limited to those defined in that RFC. For example, a field whose definition references this document can have a value that uses the Date type ({{date}}), whereas a field whose definition references RFC 8941 cannot, because it will be treated as invalid (and likely discarded) by implementations of that specification.
```

Reply:
"likely" is a bit weak; not discarding would be a spec violation, no?